discovery-provider/src/tasks/index.py

```diff
@@ -164,6 +168,20 @@ def update_ursm_address(self):
             contract_addresses["user_replica_set_manager"] = web3.toChecksumAddress(user_replica_set_manager_address)
             logger.info(f"index.py | Updated user_replica_set_manager_address={user_replica_set_manager_address}")
 
+def get_skip_tx_hash(session, redis):
+    """Fetch if there is a tx_hash to be skipped because of continuous errors
+    """
+    indexing_error = get_indexing_error(redis)
+    if isinstance(indexing_error, dict) and 'has_majority' in indexing_error and indexing_error['has_majority']:
+        skipped_tx = SkippedTransaction(
```

Comment:
maybe let's add a check to return None if count(*) SkippedTransaction > 5 or 10 or something